core/src/modules/configWriter.ts

```diff
@@ -126,23 +127,21 @@ export namespace ConfigWriter {
         }
         // Don't process objects that have missing id values.
         if (!Array.isArray(object) && !object?.id) continue;
-        const filePath = generateFilePath(
-          object as AnyConfigurationObject,
-          type
-        );
-        objects.push({ filePath, object });
-      }
-    }
 
-    const records = await GrouparooRecord.findAll();
-    const recordObjects = await Promise.all(
-      records.map((p) => p.getConfigObject())
-    );
-    if (recordObjects.length > 0) {
-      objects.push({
-        filePath: "development/records.json",
-        object: recordObjects,
-      });
+        const filePath =
+          instance instanceof GrouparooRecord
+            ? `development/${(
+                await instance.$get("model")
+              ).getConfigId()}/records.json`
```

Comment:
I feel like this could possibly live in `generateFilePath`, but here is also fine

Reply:
we went back and forth too - we left `generateFilePath ()` as a method that used the instance's ID